docs/source/noise/column_noise.rst

```diff
@@ -45,9 +45,14 @@ Leave a field blank
 
 Often some of the data in certain columns of a dataset will be missing.
 This could be because the input for that field was left blank, an answer was refused,
-or the answer was illegible or unintelligible.
-To simulate this type of noise, pseudopeople will replace the value in the relevant cell with
-:code:`numpy.nan` to indicate that the value is missing.
+or the answer was illegible or unintelligible. To simulate this type of noise, pseudopeople will 
+replace the value in the relevant cell with :code:`numpy.nan` to indicate that the value is missing. 
+
+It is important to note, however, that 
+some columns in the generated data may contain missing values, even if no noise has been added to the data.
```

Comment:
```suggestion
some columns in the generated data may contain missing values, even if no noise has been added to the data,
simply because the column is not applicable to every row.
```
Tying this in with the "not applicable" description of why values are missing.